libdd-profiling-ffi/cbindgen.toml

```diff
@@ -103,7 +103,7 @@ renaming_overrides_prefixing = true
 "Handle_EncodedProfile" = "ddog_prof_EncodedProfile"
 "Result_HandleEncodedProfile" = "ddog_prof_EncodedProfile_Result"
 
-"CancellationToken" = "struct ddog_OpaqueCancellationToken"
```

Comment:
I'm curious, did you just stumble across this in your editor or some other way? If this PR doesn't merge for whatever reason, this should probably get pulled out in merged in separately.